executor/sort.go

```diff
@@ -50,10 +51,48 @@ type SortExec struct {
 	rowPtrs []chunk.RowPtr
 
 	memTracker *memory.Tracker
```

Comment:
We need to track and show the disk usage in `explain analyze`. See https://github.com/pingcap/tidb/pull/13764 for a reference.